services/graph/pkg/service/v0/service.go

```diff
@@ -515,6 +520,39 @@ func setIdentityBackends(options Options, svc *Graph) error {
 		svc.identityBackend = options.IdentityBackend
 	}
 
+	return svc.StartListenForLogonEvents(options.Context, options.Logger)
+}
+
+func (g *Graph) StartListenForLogonEvents(ctx context.Context, l log.Logger) error {
+	if g.eventsConsumer == nil {
+		return nil
+	}
+	var _registeredEvents = []events.Unmarshaller{
+		events.UserSignedIn{},
+	}
+	evChannel, err := events.Consume(g.eventsConsumer, "graph", _registeredEvents...)
+	if err != nil {
+		l.Error().Err(err).Msg("cannot consume from nats")
+		return err
+	}
+	go func() {
+		for loop := true; loop; {
+			select {
+			case e := <-evChannel:
+				switch ev := e.Event.(type) {
+				default:
+					l.Error().Interface("event", e).Msg("unhandled event")
```

Comment:
Does it make sense to log an error here or mostly expects `events.UserSignedIn` evenst?

Reply:
It should only ever get `events.UserSignedIn` (see the `events.Consume()` line. It is really an error is anything else pops up here (not sure if that can ever happen, but better be safe than sorry)